includes/general.php

```diff
@@ -1052,7 +1052,15 @@ function pods_shortcode_run( $tags, $content = null ) {
 
 		return $pod->form( $tags['fields'], $tags['label'], $tags['thank_you'] );
 	} elseif ( ! empty( $tags['field'] ) ) {
-		if ( empty( $tags['helper'] ) ) {
+		if ( ! empty( $tags['template'] ) ) {
+			// @todo Single Pods query based on relationship field.
+			$return  = '';
+			$related = $pod->field( $tags['field'], array( 'output' => 'pods' ) );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 29d054e3102146a1da4b5c0fc77b220f</sub>

Reply:
@sc0ttkclark Is this already a requirement for 2.7.23?

Reply:
Sorry, this is for 2.8, you can ignore